### DISCUSSION TITLE
posts/2023/11/give-me-some-power/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#28] [General]
# posts/2023/11/give-me-some-power/

https://guozheng.netlify.app/posts/2023/11/give-me-some-power/

**Comment by liechi** [comment]
如果能把指甲的问题治好，不要担心花钱。身体舒适了，你可以更高效地享受学习和生活。这是非常值的投资。

前段时间有点爱听靡靡之音，折枝花满衣那种，不知道是撩动了心里哪跟弦：）

**Comment by residualsun1** [comment]
明天得去换药了，最后还是要下定决心给中指包上。身体健康，没大病，也不要有小病，其实是最奢侈的事情……

哈哈，大胆猜测你前段时间应该过得相对闲适和悠哉一些。想起以前在网易“一起听”匹配到一位上班族，ta听的重金属摇滚当时把我吓到了，很沉重，还有点黑暗的感觉，但我是觉得他压力太大，就需要这种类型的音乐释放一下。
